simtools/model/mirrors.py

```diff
@@ -136,10 +173,18 @@ def get_single_mirror_parameters(self, number):
         """
 
         def get_mirror_table_mask():
-            if isinstance(self.mirror_table["mirror_id"][0], np.str_):
-                mask = self.mirror_table["mirror_id"] == "id=" + str(number)
-            if isinstance(self.mirror_table["mirror_id"][0], np.int32):
-                mask = self.mirror_table["mirror_id"] == number
+            if isinstance(self.mirror_table["mirror_panel_id"][0], np.str_):
```

Comment:
Why are you testing against `np.str_` and not just `str`? A `np.str_` is also a `str`, but a `str` is not an `np.str_`. I think in this case you should be more inclusive.